### ISSUE TITLE
Would there be a standard list for credential_formats_supported?

### ISSUE THREAD

**Issue by Sakurann** [mattrglobal/oidc-client-bound-assertions-spec#55]
Would be confusing if w3cvp-jwt and vp-jwt and vp_jwt would be used for the same thing..?

**Comment by AdamJLemmon** [comment]
Yes we definitely need to align, DIF PE included (`jwt_vc `), on consistent naming for these things. 
Not sure the right place to have that discussion / note the decisions..

**Comment by tplooker** [comment]
Yes it would be good to have these defined in a registry and managed in IANA
